ddtrace/contrib/valkey_utils.py

```diff
@@ -0,0 +1,84 @@
+from typing import Dict
```

Comment:
Can we move this module to `ddtrace/contrib/internal/`. If we don't expect end users to directly reference these helpers we should make this module internal.